backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/MapStoreToRegion.kt

```diff
@@ -0,0 +1,15 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+import app.ehrenamtskarte.backend.regions.database.RegionEntity
+import app.ehrenamtskarte.backend.regions.database.repos.RegionsRepository
+import org.jetbrains.exposed.dao.id.EntityID
+
+fun getRegionIdFromFreinetIdOrDistrictName(projectId: EntityID<Int>, freinetId: Int?, districtName: String?): RegionEntity? {
```

Comment:
:wrench: See suggestion above. In any case, the naming is not correct as you return a whole region and not just an id.